vagrant/README.md

```diff
@@ -291,11 +438,49 @@ EOF
 ```
 And now if we go back to our [marketplace](http://192.168.33.70:8000), roughly 20% of the requests will land you on the `backend-v1` service and place the first item on sale.
 
-### 16. Let's enable Prometheus using `kumactl`:
+### Health Check
+
+The goal of Health Checks is to minimize the number of failed requests due to temporary unavailability of a target endpoint. By applying a Health Check policy you effectively instruct a dataplane to keep track of health statuses for target endpoints. Dataplane will never send a request to an endpoint that is considered "unhealthy".
 
+#### Adding Health Check Policy
+
+To apply a health check policy to backend service, run the following:
+```bash
+$ cat <<EOF | kumactl apply -f -
+type: HealthCheck
+name: frontend-to-backend
+mesh: default
+sources:
+- match:
+    service: frontend
+destinations:
+- match:
+    service: backend
+conf:
+  activeChecks:
+    interval: 10s
+    timeout: 2s
+    unhealthyThreshold: 3
+    healthyThreshold: 1
+  passiveChecks:
+    unhealthyThreshold: 3
+    penaltyInterval: 5s
+EOF
 ```
+
+### Traffic Metrics
+
+Kuma facilitates consistent traffic metrics across all dataplanes in your mesh.
+
+A user can enable traffic metrics by editing a mesh resource and providing the desired mesh-wide configuration. If necessary, metrics configuration can be customized for each Dataplane individually, e.g. to override the default metrics port that might be already in use on that particular machine. 
+
+#### Adding Traffic Metric Policy
+
+Let's enable traffic metrics by editing our mesh resource like so:
+
+```bash
 $ cat <<EOF | kumactl apply -f -
-type: Mesh
+type: mesh
```

Comment:
Let's revert this change:
* `type: mesh` is NOT valid because `Mesh` must begin with a capital `M`